src/lib/hexIdParser/hexIdParser.ts

```diff
@@ -1,3 +1,5 @@
+import midCodes from "./midCodes.json";
```

Comment:
JSON file taken from https://github.com/michaeljfazio/MIDs.  The map of country names to MID codes is in the public domain, so no licensing issues.  (See [Wikipedia](https://en.wikipedia.org/wiki/Maritime_identification_digits)

Reply:
@zackads Do you want to add this as a comment in the `midCodes.json` file? If someone new comes onto the project they might question where we took the JSON file from.

Adding a comment in this case is okay for me, otherwise it's not clear where we have taken this from.

Thoughts?

Reply:
We discussed a README.md in the folder this morning.  I did this, but feel it's overkill.  Instead I've pushed a tsdoc comment on the method:

```typescript
  /**
   * Translates a country code (MID) into the corresponding country name.
   *
   * @remarks Maritime Identification Digits (MIDs) are an international
   * standard maintained by the International Telecommunications Union in the
   * public domain.  The lookup table is defined in midCodes.json, borrowed from
   * https://github.com/michaeljfazio/MIDs/blob/master/mids.json for
   * convenience.
   *
   * The authoritative source for MID codes is
   * https://www.itu.int/en/ITU-R/terrestrial/fmd/Pages/mid.aspx).
   *
   * @param hexId - The hexId string
   * @returns A string of the country name encoded in the hexId
   *
   */
  public static countryName(hexId: string): string {
    return midCodes[this.countryCode(hexId).toString()][3];
  }

What do you reckon @matthew-a-carr ?
```